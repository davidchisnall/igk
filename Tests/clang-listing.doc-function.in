\docfile{includes/test.cc}
\functiondoc{main}
