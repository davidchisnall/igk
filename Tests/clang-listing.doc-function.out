
\clang-doc[code-declaration-entity=main,code-declaration-kind=function]{\code[code-kind=listing]{\code-run[token-kind=TypeRef]{int} \code-run[token-kind=FunctionName]{main}\code-run[token-kind=Punctuation]{(}\code-run[token-kind=TypeRef]{int} \code-run[token-kind=ParamName]{argc}\code-run[token-kind=Punctuation]{,} \code-run[token-kind=TypeRef]{char **} \code-run[token-kind=ParamName]{argv}\code-run[token-kind=Punctuation]{)}}\p{ This is the \code-run{main} function!  Yes it is!}\p{ MAAAAAIIIIN!}\itemize{\item{ Is it?}\item{ Yes, it is!}\item{ It takes \code-run{argc} and \code-run{argv} as arguments.}}}
